docs/source/tutorials/parsl-rp.ipynb

```diff
@@ -231,11 +209,14 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "id": "6a6f2135",
    "metadata": {},
    "source": [
-    "Create a simple Parsl `@bash_app` to invoke the `NWChem` task. The `bash_app` requires the type of the task which in this case it is an `MPI` and the number of cpu_processes (2 MPI ranks where each rank takes 1 core) to run on."
+    "Create a simple Parsl `@bash_app` to invoke the `NWChem` task. The `bash_app` requires the type of the task and the number of `cpu_processes` on which to run. In this case, the type of the task is `MPI`, and the number of `cpu_processes` is 2 `MPI` ranks, where each rank takes 1 core.\n",
+    "\n",
+    "Once the `bash_app` (executable task) is invoked, the RadicalPilotExecutor submits the task to the runtime system and waits for it. RadicalPilotExecutor creates a designated `sandbox` folder that contains the tasks and their `stdout/stderr` files that contain after-execution task output or errors. "
```

Comment:
"and waits for it" -> and wait for them to be executed
Delete " that contain after-execution task output or errors."